client.go

```diff
@@ -10,7 +11,7 @@ import (
 	"git.apache.org/thrift.git/lib/go/thrift"
 )
 
-type ExtensionManager interface {
+type ExtensionManagerClientI interface {
```

Comment:
naming interfaces with `I` is not a pattern we want to adopt IMO, it's hevily discouraged in Go.

Reply:
I made this change to try to get `mockimpl` to work, but it's broken anyway so I might as well change it back.